sdk/identity/azure-identity/azure/identity/_credentials/authorization_code.py

```diff
@@ -86,7 +86,12 @@ def get_token(
           ``response`` attribute.
         """
         # pylint:disable=useless-super-delegation
-        return super(AuthorizationCodeCredential, self).get_token(*scopes, claims=claims, tenant_id=tenant_id, **kwargs)
+        client_secret = (
+            kwargs.pop("client_secret", None) or self._client_secret
+        )  # If secret not provided in method, use init secret
+        return super(AuthorizationCodeCredential, self).get_token(
+            *scopes, claims=claims, tenant_id=tenant_id, client_secret=client_secret, **kwargs
```

Comment:
We also need to pass in client_secret in https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/azure/identity/_credentials/authorization_code.py#L96

Reply:
And https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/azure/identity/_credentials/authorization_code.py#L105

Reply:
Well. Never mind. if we put it in get_token, we will have it in the kwarg of _request_token.